app/src/main/java/com/babylon/wallet/android/domain/usecases/transaction/PollTransactionStatusUseCase.kt

```diff
@@ -2,17 +2,21 @@ package com.babylon.wallet.android.domain.usecases.transaction
 
 import com.babylon.wallet.android.data.dapp.model.TransactionType
 import com.babylon.wallet.android.data.repository.TransactionStatusData
-import com.babylon.wallet.android.data.repository.transaction.TransactionRepository
+import com.babylon.wallet.android.di.coroutines.IoDispatcher
 import com.babylon.wallet.android.domain.RadixWalletException
 import com.babylon.wallet.android.utils.toMinutes
 import com.radixdlt.sargon.Epoch
 import com.radixdlt.sargon.TransactionIntentHash
 import com.radixdlt.sargon.TransactionStatus
 import com.radixdlt.sargon.TransactionStatusReason
+import com.radixdlt.sargon.os.SargonOsManager
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.withContext
 import javax.inject.Inject
 
 class PollTransactionStatusUseCase @Inject constructor(
```

Comment:
same for this if you decide to rename the `PollPreAuthorizationStatusUseCase` :)